docs/articles/licensing/publishing-binaries.rst

```diff
@@ -0,0 +1,30 @@
+Publishing binary images
+========================== 
+
+We release binary images of our products that are licensed under the 
+[https://www.gnu.org/licenses/old-licenses/gpl-2.0.html](GNU Public License).
+This is considered "distribution" under the GPL. When distributing binaries, 
+or object code, covered under the GPL "complete and corresponding" source code 
+is required to be provided as well. We comply with these obligations 
+by publishing source code on GitHub and through our continuous integration 
+system (CI).
+
+In order to comply with all FOSS software licenses, public CI systems must 
+plublish complete and corresponding source code in conjunction with the publishing
+of object code. This include a yocto sstatecache, binary images for hardware 
```

Comment:
Actually, there is a way in yocto to generate a `SOURCE_MIRROR`, which we should probably use when doing a release. With such a mirror, one could reproduce a build without having network access at all - meaning that it will contain complete and corresponding source code :-)

Reply:
The thing is that in some cases the source code needs to be potentially delivered to the user for years after the release, so the "problem" we have is that we don't want to trigger the clause at all too often.

Reply:
Of course, I just meant that instead of saving the sstate cache, we could create a source mirror, whenever the clause is triggered.

Reply:
I prefer to use the yocto download dir but I guess a source mirror is good to. It might speed up builds actually.

Reply:
@jeremiah Are you sure using the download dir yields EXACTLY the same result though? I mean the very idea of a SOURCE_MIRROR is so that you can build without any network access (ignoring AUTOREV recipies, of course).

Reply:
It should be possible to tie this together with the reproducibility use-cases in general which are part of releasing. If we can use the same archive to provide sources as we use to reproduce releases then I think we should align this with https://github.com/Pelagicore/software-factory-blueprint/pull/23